### PR TITLE
Add testID to ActionSheet

### PR DESCRIPTION
**Pull request by M-i-k-e-l**

### CODE REVIEW

File: src/components/actionSheet/index.js [hunk 1]
```diff
@@ -159,7 +159,7 @@ export default class ActionSheet extends BaseComponent {
 
   renderSheet() {    
     return (
-      <View bg-white>
+      <View testID={this.props.testID} bg-white>
```
Comment (ethanshar) [inline src/components/actionSheet/index.js]: the task was to add testID to the action sheet options (:

Comment (ethanshar) [reply src/components/actionSheet/index.js]: @M-i-k-e-l 
I remember talking about this PR, is it possible you didn't push the changes?

Comment (M-i-k-e-l) [reply src/components/actionSheet/index.js]: I did push it, not sure why this has not changed but the [code](https://github.com/wix/react-native-ui-lib/pull/418/files) has changed

Comment (ethanshar) [reply src/components/actionSheet/index.js]: Look at - "File Changed" tab. 
There's only one file that has changed, weird..

Comment (M-i-k-e-l) [reply src/components/actionSheet/index.js]: I did not push the testID input in the screen, should I have?

**Pull request merged**